src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -2808,10 +2808,20 @@ public BlittableJsonReaderObject Read<T>(TransactionOperationContext<T> context,
             }
         }
 
-        public static unsafe BlittableJsonReaderObject ReadInternal<T>(TransactionOperationContext<T> context, out long etag, Slice key)
+        public static unsafe BlittableJsonReaderObject ReadInternal<T>(TransactionOperationContext<T> context, out long etag, Slice key, bool ignoreMissingTable = false)
             where T : RavenTransaction
         {
             var items = context.Transaction.InnerTransaction.OpenTable(ItemsSchema, Items);
+            if (items == null && ignoreMissingTable)
```

Comment:
Why aren't we checking and then creating the tables, then?

Reply:
We do that after the _Schema Upgrade_.

Reply:
Why not simply always return `etag = 0` and `null` when items are null instead of introducing a parameter?

Reply:
done